runtimes/peregrine/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["KILT <info@kilt.io>"]
 edition = "2018"
-name = "kilt-parachain-runtime"
+name = "peregrine-runtime"
```

Comment:
This change might also effect the Polkadot Apps in regards to the types.

Reply:
just checked. Spiritnet runtime still works